.github/policies/labelManagement.issueOpened.yml

```diff
@@ -15,6 +15,13 @@ configuration:
               action: Opened
         then:
           - addCodeFlowLink
+          # If the user is a first-time contributor, add the Needs-CLA Label
+          - if:
+              - activitySenderHasAssociation:
+                  association: FIRST_TIME_CONTRIBUTOR
```

Comment:
Is there an index or listing of the available terms?

Reply:
Yes, and this is one of the allowed values. The GitOps team is working on a patch

Reply:
Actually, after reviewing, the patch should be live. I'll pull in master and see if the issue persists